app/models/concerns/user_permissions.rb

```diff
@@ -178,11 +179,16 @@ def can_delete_projects_or_data_sources
     end
 
     def can_access_some_cohorts
-      can_manage_cohorts? || can_edit_cohort_clients? || can_edit_assigned_cohorts? || can_view_assigned_cohorts?
+      can_edit_some_cohorts || can_view_cohorts? || can_view_inactive_cohort_clients?
```

Comment:
I don't think you'd ever have `can_view_inactive_cohort_clients?` without `can_edit_some_cohorts` or `can_view_cohorts?` maybe the following is sufficient?
```suggestion
      can_edit_some_cohorts? || can_view_cohorts?
```

Reply:
I agree -- being able to see the inactive clients, but not the current cohort makes no sense.